### PR TITLE
Update [2] ITEM.tsv

### PR DESCRIPTION
**Pull request by Flevalt**
Quick Question: 
Should we only mark Edits with "x" or also Korean -> English translations?

### CODE REVIEW

**Comment by Sourpusss** [comment]
If you feel them to be accurate, you should mark all edited lines with "x." Also, after you do this with this request, you should add them to your earlier pull.

Looking at your translations, though, you shouldn't put x's on the majority of them.  A lot of them require editing still.  Just mark the ones that make absolute sense, and you would understand if you came across in-game.

**Comment by Sunaries** [comment]
Replace: "it can be obtained from..." with "Obtained from". Shorter and sounds better. 
From checking the lines, it looks like you used Google Translate?

I left some notes. There are many more incorrect strings, but I have to go now.

File: ITEM.tsv [hunk 1]
```diff
@@ -2875,15 +2875,15 @@ ITEM_20150317_002874	Scroll of Spatial Distortion
 ITEM_20150317_002875	A spell scroll that opens a distortion in the spatial fabric. Here you can obtain the key that belonged to the Great King.
 ITEM_20150317_002876	Great King's Key
 ITEM_20150317_002877	A key used for releasing the seal of Great King Zachariel.
-ITEM_20150317_002878	
+ITEM_20150317_002878	Leather Geurol
 ITEM_20150317_002879	One of the task requirements by the Hoplite Master. Looks extremely sturdy.
-ITEM_20150317_002880	
+ITEM_20150317_002880	Stinger of Chaiporo
```
Comment (Sunaries) [inline ITEM.tsv:2880]: Chafperor's Poison Dart

File: ITEM.tsv [hunk 2]
```diff
@@ -2875,15 +2875,15 @@ ITEM_20150317_002874	Scroll of Spatial Distortion
 ITEM_20150317_002875	A spell scroll that opens a distortion in the spatial fabric. Here you can obtain the key that belonged to the Great King.
 ITEM_20150317_002876	Great King's Key
 ITEM_20150317_002877	A key used for releasing the seal of Great King Zachariel.
-ITEM_20150317_002878	
+ITEM_20150317_002878	Leather Geurol
```
Comment (Sunaries) [inline ITEM.tsv:2878]: Groll's Leather

File: ITEM.tsv [hunk 3]
```diff
@@ -2875,15 +2875,15 @@ ITEM_20150317_002874	Scroll of Spatial Distortion
...
 ITEM_20150317_002879	One of the task requirements by the Hoplite Master. Looks extremely sturdy.
-ITEM_20150317_002880	
+ITEM_20150317_002880	Stinger of Chaiporo
 ITEM_20150317_002881	One of the task requirements by the Hoplite Master. It would be fatal to be stabbed by it.
-ITEM_20150317_002882	
+ITEM_20150317_002882	Black Powder of Yuko Foods
```
Comment (Sunaries) [inline ITEM.tsv:2882]: Ukopus' Black Powder

File: ITEM.tsv [hunk 4]
```diff
@@ -2875,15 +2875,15 @@ ITEM_20150317_002874	Scroll of Spatial Distortion
...
 ITEM_20150317_002883	One of the task requirements by the Hoplite Master. It scatters at the slightest breath.
 ITEM_20150317_002884	Ellum's Sticky Sap
 ITEM_20150317_002885	Quest item.
-ITEM_20150317_002886	
+ITEM_20150317_002886	Center of a flower of Meshumu
```
Comment (Sunaries) [inline ITEM.tsv:2886]: Matsum's Flower

File: ITEM.tsv [hunk 5]
```diff
@@ -2891,7 +2891,7 @@ ITEM_20150317_002890	Material requested by the Linker Master. He said he needed
 ITEM_20150317_002891	Beliggler's Claw
 ITEM_20150317_002892	Sapper Master's Hidden Goods
 ITEM_20150317_002893	The Sapper Master requested these goods to be returned to their original owner. Did he steal these?
-ITEM_20150317_002894	
+ITEM_20150317_002894	Merogu Leather
```
Comment (Sunaries) [inline ITEM.tsv:2894]: Merog's Leather

File: ITEM.tsv [hunk 6]
```diff
@@ -2905,10 +2905,10 @@ ITEM_20150317_002904	Goods for Offering
 ITEM_20150317_002905	Items to be offered to the Goddess Statue.
 ITEM_20150317_002906	Priest Master's Holy Water of Purification
 ITEM_20150317_002907	Holy Water used for banishing evil spirits. A very pure water.
-ITEM_20150317_002908	
+ITEM_20150317_002908	Skin of Grohl
```
Comment (Sunaries) [inline ITEM.tsv:2908]: Groll's Skin

File: ITEM.tsv [hunk 7]
```diff
@@ -2905,10 +2905,10 @@ ITEM_20150317_002904	Goods for Offering
...
 ITEM_20150317_002909	Materials for the Wizard Master's research.
-ITEM_20150317_002910	
-ITEM_20150317_002911	
+ITEM_20150317_002910	Antennae of Chaiporo
```
Comment (Sunaries) [inline ITEM.tsv:2910]: Chafperor's Feeler

File: ITEM.tsv [hunk 8]
```diff
@@ -2905,10 +2905,10 @@ ITEM_20150317_002904	Goods for Offering
...
+ITEM_20150317_002911	Yuko Foods Nucleus
```
Comment (Sunaries) [inline ITEM.tsv:2911]: Ukopus Core

File: ITEM.tsv [hunk 9]
```diff
@@ -2925,19 +2925,19 @@ ITEM_20150317_002924	Strange Map
...
 ITEM_20150317_002933	A piece of cloth probably from the outfit of the Tool Shop Merchant's Father. You can feel great effort from the needlepoints.
 ITEM_20150317_002934	Box of Food for Stray Cat
 ITEM_20150317_002935	A box containing a delicacy liked by the stray cats. Used to catch stray cats. It has a fishy smell.
 ITEM_20150317_002936	Sun Essense
 ITEM_20150317_002937	Spirit of fire collected into a pot.
-ITEM_20150317_002938	
+ITEM_20150317_002938	Cockatrice Beak
 ITEM_20150317_002939	Material requested by the Cryomancer Master as an alternative payment for his teaching fees.
-ITEM_20150317_002940	
+ITEM_20150317_002940	Armadillo Scales
```
Comment (Sunaries) [inline ITEM.tsv:2940]: Arma's Scale

File: ITEM.tsv [hunk 10]
```diff
@@ -3274,13 +3274,13 @@ ITEM_20150317_003273	Silver Dye for Magical Tattoo
 ITEM_20150317_003274	Silver dye used for Simon Shaw's tattoo. It is relatively easy to get.
 ITEM_20150317_003275	Cordelier's Parchment
 ITEM_20150317_003276	Parchment Cordelier gave you. Although coal powder has been used, it is hard to see the letters because the parchment has darkened.
-ITEM_20150317_003277	
-ITEM_20150317_003278	
+ITEM_20150317_003277	Nucleus of the platform Nuremberg Rem
```
Comment (Sunaries) [inline ITEM.tsv:3277]: Fallenlem Core

File: ITEM.tsv [hunk 11]
```diff
@@ -3274,13 +3274,13 @@ ITEM_20150317_003273	Silver Dye for Magical Tattoo
...
+ITEM_20150317_003278	Power source to move the Porurunremu. It is necessary for the other case can not write brooch.
```
Comment (Sunaries) [inline ITEM.tsv:3278]: Contains a power source that moves the Fallenlem. Grita, who can't use magic, needs it.

File: ITEM.tsv [hunk 12]
```diff
@@ -3274,13 +3274,13 @@ ITEM_20150317_003273	Silver Dye for Magical Tattoo
...
 ITEM_20150317_003279	Flame Crystal
 ITEM_20150317_003280	Needed to stabilize the Flame Gem.
-ITEM_20150317_003281	
+ITEM_20150317_003281	Nudoku Kochi's Body Fluids
```
Comment (Sunaries) [inline ITEM.tsv:3281]: Bagworm's Body Fluid

File: ITEM.tsv [hunk 13]
```diff
@@ -3274,13 +3274,13 @@ ITEM_20150317_003273	Silver Dye for Magical Tattoo
...
 ITEM_20150317_003282	Material needed to make a Bokor doll.
-ITEM_20150317_003283	
+ITEM_20150317_003283	Stem of Donboru
```
Comment (Sunaries) [inline ITEM.tsv:3283]: Thornball's Stem

File: ITEM.tsv [hunk 14]
```diff
@@ -3359,10 +3359,10 @@ ITEM_20150317_003358	Enhanced Thorn Flower Stimulant
 ITEM_20150317_003359	It has a better effect than the Thorn Flower Powder 
 ITEM_20150317_003360	Thorn Flower Fluid
 ITEM_20150317_003361	It has an awakening effect if mixed with a Merog's spit.
-ITEM_20150317_003362	
-ITEM_20150317_003363	
-ITEM_20150317_003364	
-ITEM_20150317_003365	
+ITEM_20150317_003362	Slime Liquid
```
Comment (Sunaries) [inline ITEM.tsv:3362]: Slime's Body Liquid

File: ITEM.tsv [hunk 15]
```diff
@@ -3359,10 +3359,10 @@ ITEM_20150317_003358	Enhanced Thorn Flower Stimulant
...
+ITEM_20150317_003363	Material necessary to prove Elemental master list qualification
```
Comment (Sunaries) [inline ITEM.tsv:3363]: Material requested by the Elementalist Master to prove your qualification

File: ITEM.tsv [hunk 16]
```diff
@@ -3359,10 +3359,10 @@ ITEM_20150317_003358	Enhanced Thorn Flower Stimulant
...
+ITEM_20150317_003364	Hollow Bone Fragments Vendor
```
Comment (Sunaries) [inline ITEM.tsv:3364]: Hallowventer's Bone Piece

File: ITEM.tsv [hunk 17]
```diff
@@ -3359,10 +3359,10 @@ ITEM_20150317_003358	Enhanced Thorn Flower Stimulant
...
+ITEM_20150317_003365	Driving Nucleus of the Cemetery Golem
```
Comment (Sunaries) [inline ITEM.tsv:3365]: Graveyard Golem's 구동핵

File: ITEM.tsv [hunk 18]
```diff
@@ -3407,10 +3407,10 @@ ITEM_20150317_003406	Moya Body Fluid Sample
 ITEM_20150317_003407	A body fluid sample extracted from a Moya
 ITEM_20150317_003408	Intermediate Arrow Sample
 ITEM_20150317_003409	An arrow sample created by the Fletcher Master. It has good balance.
-ITEM_20150317_003410	
+ITEM_20150317_003410	Sounding Wood
 ITEM_20150317_003411	Wood that the Fletcher Master is looking for. It produces a fine timbre when knocked on.
-ITEM_20150317_003412	
-ITEM_20150317_003413	
+ITEM_20150317_003412	Kojirukojiru Tree
```
Comment (Sunaries) [inline ITEM.tsv:3412]: Grime-Encrusted Tree

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**